includes/core/classes/class-setup.php

```diff
@@ -82,6 +82,8 @@ protected function instantiate_classes(): void {
 	 * @return void
 	 */
 	protected function setup_hooks(): void {
+		add_action( 'init', array( $this, 'load_gatherpress_textdomain' ), 0 );
```

Comment:
Why `0`? I try not to be arbitrary with priorities and add a comment when the priority is not `10` so the next person knows the reason why the priority is the way it is. Usually, I try not to deviate far from `10` (likely to be `9` or `11` if it creates an issue with another priority that is the default `10`). If I want it to be the first or last thing run, I'll usually use `PHP_INT_MIN` or `PHP_INT_MAX`, but I try to not be too far off the thing that is keeping me from setting the priority to `10`.

Reply:
Hmmm ... why did I do this here ... I need to have a look again.

Thanks for bringing this up! I often do the same and add comments about non-10 priorities for my future-self, but must have forgotten here.

Reply:
_This will be solved_, **when we accept turning the plugin upside-down and changing the load order.** 

OMG!

We need to have a :coffee: together @mauteri , I think.